users/views.py

```diff
@@ -60,4 +62,25 @@ def post(self, requst):
             return JsonResponse({'message' : 'Key Error'}, status = 401)
         
         except ValidationError as error:
-            return JsonResponse({'message' : error.message}, status = 400)
\ No newline at end of file
+            return JsonResponse({'message' : error.message}, status = 400)
+
+class SignInView(View):
+    def post(self, request):
+        try:
+            data = json.loads(request.body)
+            username = data['username']
+            password = data['password']
+            user = User.objects.get(username = username)
+
+            if not bcrypt.checkpw(password.encode('utf-8') , user.password.encode('utf-8')):
+                return JsonResponse({"message" : "Invalid User"}, status = 401)
+
+            token = jwt.encode({'user_id': user.id}, SECRET_KEY, ALGORITHM)
+
+            return JsonResponse({'token' : token}, status = 200)
+        
+        except KeyError:
+            return JsonResponse({'message' : 'Key Error'}, status = 401)
```

Comment:
status code  확인해 주세요.

```suggestion
        except KeyError:
            return JsonResponse({'message' : 'Key Error'}, status = 400)
```